src/components/Header.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react'
+import MoviesList from './MoviesList'
+import MoviesData from '../data/moviesList'
+import Search from './Search'
+import SortSearch from './SortSearch'
+import SortByCategory from './SortByCategory'
+
+class Header extends Component {
+    constructor(props) {
+        super(props);
+        this.MoviesData = MoviesData.data;
```

Comment:
better to put such kind of data to state

Reply:
and do not keep it in class

Reply:
Updated